packages/itwinui-react/src/core/FileUpload/FileUploadCardAction.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import cx from 'classnames';
+
+export type FileUploadCardActionProps = {
+  /**
+   * Buttons in the dialog bar.
+   */
+  children: React.ReactNode;
+} & React.ComponentPropsWithRef<'div'>;
+
+export const FileUploadCardAction = React.forwardRef<
+  HTMLDivElement,
+  FileUploadCardActionProps
+>((props, ref) => {
+  const { children, className, ...rest } = props;
+  return (
+    <div className={cx('iui-file-card-action', className)} ref={ref} {...rest}>
+      <label className='iui-anchor'>{children}</label>
```

Comment:
user needs to have control over the `<label>` element too, so it should probably be another subcomponent

Reply:
This new subcomponent has been added